### ISSUE TITLE
Integrate OME-Zarr and a visualisation for this data

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy#16862]
The spec:  https://github.com/ome/ngff
A python library: https://github.com/ome/ome-zarr-py
A viewer: https://github.com/hms-dbmi/vizarr

**Comment by beatrizserrano** [comment]
OME-Zarr converter and data transfer were developed within Euro-BioImaging but not available in the ToolShed because of missing dependencies in conda-forge. Let's take this discussion to the BioHackathon :)

**Comment by bgruening** [comment]
Isn't this what is needed for the converters? https://anaconda.org/bioconda/bftools

**Comment by bugraoezdemir** [comment]
The converter in the `bftools` package, unfortunately, cannot convert to OME-Zarr. 

Currently, the most commonly-used tool to convert various proprietary image formats to OME-Zarr is `bioformats2raw`: 

https://anaconda.org/ome/bioformats2raw 
https://github.com/glencoesoftware/bioformats2raw

`Bioformats2raw` is, however, not registered on conda-forge or bioconda. Previously, we had a discussion on moving it to conda-forge and there was some progress, but then it has been on stall: 

https://github.com/ome/conda-bioformats2raw/issues/14 

Perhaps we should resume this effort now?

**Comment by beatrizserrano** [comment]
ping @annefou

**Comment by davelopez** [comment]
Prototype integrating Zarr and OME-Zarr datatypes is currently testable in https://github.com/galaxyproject/galaxy/pull/19040

PR adding Vizarr as a visualization plugin coming soon (need to fix some issues serving "dot files")

**Comment by annefou** [comment]
Really super cool! 
I just installed in my local instance and it is awesome! We will work on adding GeoZarr (https://github.com/zarr-developers/geozarr-spec) too! Thanks.

**Comment by bgruening** [comment]
I think this one can be closed. @davelopez can you confirm?

**Comment by davelopez** [comment]
Yeah, specific improvements or bugs can be addressed in other issues, thanks for the reminder 👍
